syntax="proto3";
package module1;

option cc_generic_services = true;

message EchoRequest {
      string message = 1;
};

message EchoResponse {
      string message = 1;
};

service EchoService {
      rpc Echo(EchoRequest) returns (EchoResponse);
};